_pages/bea/current.md

```diff
@@ -262,25 +262,25 @@ More information can be found on the task [webpage](https://www.cl.cam.ac.uk/res
                      
                     <li><i>Grammatical-Error-Aware Incorrect Example Retrieval System for Learners of Japanese as a Second Language</i>. Mio Arai, Masahiro Kaneko and Mamoru Komachi. </li>
                      
-                    <li><i>Automated Content Feedback Generation for Non-native Spontaneous Speech</i>. Su-Youn Yoon, Ching-Ni Hsieh, Klaus Zechner, Matthew Mulholland and Yuan Wang. </li>
+                    <li><i>Toward Automated Content Feedback Generation for Non-native Spontaneous Speech</i>. Su-Youn Yoon, Ching-Ni Hsieh and Klaus Zechner. </li>
```

Comment:
@ekochmar this particular edit looked strange to me so I checked with Su-Youn and she confirmed that she did not change the author list in her camera-ready submission – just the title. Can you confirm if this is the case?

Reply:
Sorry about that. The correct entry should be: <li><i>Toward Automated Content Feedback Generation for Non-native Spontaneous Speech</i>. Su-Youn Yoon, Ching-Ni Hsieh, Klaus Zechner, Matthew Mulholland, Yuan Wang, and Nitin Madnani </li> Would you be able to add this change?

Reply:
Done.